nushell: 0.107.0 -> 0.108.0

shgew opened the pull request:
Diffs: 
 * https://github.com/nushell/nushell/compare/0.107.0...0.108.0
 * https://github.com/fmotalleb/nu_plugin_desktop_notifications/compare/v0.106.1...v0.108.0
 * https://github.com/cptpiepmatz/nu-plugin-highlight/compare/v1.4.8+0.106.0...v1.4.10+0.108.0
 * https://github.com/idanarye/nu_plugin_skim/compare/v0.19.0...v0.20.1
 * https://github.com/Yethal/nu_plugin_hcl/compare/0.107.0...0.108.0
 * https://github.com/abusch/nu_plugin_semver/compare/v0.11.7...v0.11.8
 
 The rest of the non-native plugins are either not yet updated or marked as broken.

<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin (only supported packages)
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [x] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

shgew committed:
nushell: 0.107.0 -> 0.108.0

Diff: https://github.com/nushell/nushell/compare/0.107.0...0.108.0

shgew committed:
nushellPlugins.desktop_notifications: 0.106.1 -> 0.108.0

Diff:
https://github.com/FMotalleb/nu_plugin_desktop_notifications/compare/v0.106.1...v0.108.0

shgew committed:
nushellPlugins.skim: 0.19.0 -> 0.20.1

Diff: https://github.com/idanarye/nu_plugin_skim/compare/v0.19.0...v0.20.1

shgew commented:
> Why are you upgrading the plugins to a version that is not compatible with 0.108.0.
> 
> Nushell 0.108.0 desktop_notification 0.107.0

Because that was the latest plugin version available at the time. I see that the plugin now has a new version. I will upgrade it to that instead.

VuiMuich commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 452687`
Commit: `904611e9ce8f67f8a79d6e69438ea1e28bec5610`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 9 packages built:</summary>
  <ul>
    <li>nushell</li>
    <li>nushellPlugins.desktop_notifications</li>
    <li>nushellPlugins.formats</li>
    <li>nushellPlugins.gstat</li>
    <li>nushellPlugins.polars</li>
    <li>nushellPlugins.query</li>
    <li>nushellPlugins.skim</li>
    <li>tests.pkgs-lib</li>
    <li>tests.writers.simple.nu</li>
  </ul>
</details>

shgew commented:
> @shgew I just pushed 0.108.0 tag for nu_plugin_hcl, can you add that to the pr?

Sure thing. Thanks for the info. I will add that to my todos today.

shgew committed:
nushellPlugins.hcl: 0.107.0 -> 0.108.0

Diff: https://github.com/Yethal/nu_plugin_hcl/compare/0.107.0...0.108.0

shgew committed:
nushellPlugins.highlight: 1.4.8+0.106.0 -> 1.4.10+0.108.0

Diff: https://github.com/cptpiepmatz/nu-plugin-highlight/compare/v1.4.8+0.106.0...v1.4.10+0.108.0

shgew committed:
nushellPlugins.semver: 0.11.7 -> 0.11.8

Diff: https://github.com/abusch/nu_plugin_semver/compare/v0.11.7...v0.11.8

Changelog: https://github.com/abusch/nu_plugin_semver/blob/v0.11.8/CHANGELOG.md

shgew commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 452687`
Commit: `c5dc09908f842d8f5e6a8a67491e4cd2725b4a39`

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 9 packages built:</summary>
  <ul>
    <li>nushell</li>
    <li>nushellPlugins.formats</li>
    <li>nushellPlugins.gstat</li>
    <li>nushellPlugins.hcl</li>
    <li>nushellPlugins.highlight</li>
    <li>nushellPlugins.polars</li>
    <li>nushellPlugins.query</li>
    <li>nushellPlugins.skim</li>
    <li>tests.writers.simple.nu</li>
  </ul>
</details>

koffydrop reviewed (approved):
Approved automatically following the successful run of `nixpkgs-review`.

4evy reviewed (approved):
Approved automatically following the successful run of `nixpkgs-review`.
